doc/source/contributing/development_plan.rst

```diff
@@ -1,45 +1,51 @@
+Please :ref:`contact us<contact_ref_label>` if you have any questions or would like some guidance to get involved!
 
 Data Analysis and Interaction
------------------------------
-Multiple forms of traditional and advanced computational analysis are used by researchers to probe and analyze large datasets to answer challenging questions about the systems the data describes. These analysis techniques include filtering, application of corrections, trend detection, feature detection, statistics, and machine learning, among others. icepyx aims to easily integrate existing libraries that specialize in these types of analysis by providing easy ways to manipulate ICESat-2 data into the appropriate form required by each library and showcasing the use of these complex analyses to answer glaciological questions through easily-modifiable example workflows based on actual use cases.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Many data analysis techniques (filtering, corrections, trend detection, feature detection, statistics, machine learning, etc.)
+are used by researchers to analyze ICESat-2 data products.
+As part of the broader Python ecosystem, relevant libraries that specialize in these techniques are easily incorporated into a single work environment.
+In addition, creating ICESat-2 specific extensions for Xarray and Pandas data structures will enhance our ability to utilize these analysis tools
+by providing easy ways to manipulate ICESat-2 data in the appropriate input type required by each library.
+Workflows showcasing complex analyses to answer pressing science questions provide an opportunity for new reseachers to build on existing work.
 
 Validation and Integration with Other Products
-----------------------------------------------
-The complexity of multiple data access systems, many with different metadata formats and API access types, presents a challenge for finding and integrating diverse datasets to construct long time series. Many open-source resources provide tools for manipulating certain types of datasets, but few collate these resources into one computing environment (see `Improving Accessibility to Advanced Computing`_) and provide wrappers and examples to easily conduct frequently performed analysis tasks. This portion of the development plan, driven by researcher use cases, will combine existing resources with new ones to improve researcher ability to easily compare diverse datasets across varying sensor types and spatial and temporal scales.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The complexity of multiple data access systems, many with different metadata formats and API access types, 
+presents a challenge for finding and integrating diverse datasets. 
+Driven by researcher use cases, icepyx contains a framework for easily adding a new product/sensor to an existing data analysis pipeline,
+improving researcher ability to easily compare diverse datasets across varying sensor types and spatial and temporal scales.
```

Comment:
Debating on whether to take out the 'easily' adjective, when has data integration ever been easy :slightly_smiling_face:

Reply:
maybe instead  '...icepyx contains a consistent framework for adding a new product/sensor.....' because I think the key here is that icepyx can enable downloads of many types of data, and it's easy because it's consistent and in one place.